packages/spaces/src/lib/SpacesGhostText.tsx

```diff
@@ -0,0 +1,23 @@
+import { useSpaces } from './Spaces';
+
+export type SpacesGhostTextProps = {
+  spaceId: string;
+  id?: string;
+  [key: string]: any;
```

Comment:
I think we can make this a little more accurate by doing something like this
```ts
export type SpacesGhostTextProps = {
   // ....
} & JSX.IntrinsicAttributes['small']
```

I believe that will allow anything we can pass to `small`. The name of the type might be a little different.

Reply:
fixed